readthedocs/settings/testdocsitalia.py

```diff
@@ -13,10 +13,19 @@
     }
 }
 
+CLASS_OVERRIDES = {
```

Comment:
To fix tests I'd avoid these changes here and add them with override_settings on our own tests

Reply:
yes, that was a last ditch-attempt 
settings_override does not work with `CLASSES_OVERRIDES` anyway, thus to test the resolver I think the best way is just to test that specific method